easybuild/easyconfigs/p/PEAR/PEAR-0.9.10-iccifort-2017.4.196-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'PEAR'
+version = '0.9.10'
+
+homepage = 'http://sco.h-its.org/exelixis/web/software/pear/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'iccifort', 'version': '2017.4.196-GCC-6.4.0-2.28'}
+
+# The source archive must now be requested at
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
```

Comment:
@vanzod Should be `'%(namelower)s-src-%(version)s.tar.gz'`?

Not sure how the test worked for you with this in place...